blocks/cards/cards.js

```diff
@@ -10,7 +10,29 @@ export default function decorate(block) {
     while (row.firstElementChild) li.append(row.firstElementChild);
     [...li.children].forEach((div) => {
       if (div.children.length === 1 && div.querySelector('picture')) div.className = 'cards-card-image';
-      else div.className = 'cards-card-body';
+      else { div.className = 'cards-card-body'; }
+
+      /* cards simple */
+      if (block.classList.contains('cards') && block.classList.contains('simple')) {
+        const anchorEl = div.querySelector('a');
+        if (anchorEl) {
+          anchorEl.textContent = '';
+          div.append(anchorEl);
+          div.querySelector('.button-container').remove();
+        }
+        const cardBodyContent = document.createElement('div');
+        cardBodyContent.className = 'card-body-content';
```

Comment:
Is this container necessary? Why not just have the structure as `li > a > div.cards-card-body`?

Reply:
This was intentional because I need to add animated styling as hover effect. Adding that directly to card body is throwing up because of paddings and other styling. This way it is clean and easy to handle the transitions of hover.